remote/remote_state_gcs.go

```diff
@@ -52,6 +52,7 @@ var terragruntGCSOnlyConfigs = []string{
 type RemoteStateConfigGCS struct {
 	Bucket        string `mapstructure:"bucket"`
 	Credentials   string `mapstructure:"credentials"`
+	AccessToken   string `mapstructure:"access_token"`
```

Comment:
I think added field should be documented in GCS bucket reference `docs/_docs/04_reference/config-blocks-and-attributes.md`

Reply:
done